docs/deploy-custom-image.rst

```diff
@@ -4,90 +4,114 @@
 Deploying custom Docker images
 ==============================
 
-Scrapy Cloud 2.0 allows you to run your spiders in custom Docker containers. The tools for managing and deploying Docker images are bundled under ``shub image``.
+.. note::
 
-Deployment
-==========
-This section describes how to build and deploy to Scrapy Cloud 2.0 a custom Docker image for a Scrapy project.
-
-.. _step-one:
+    This feature is currently only available for paying customers.
 
-1. Initialization
------------------
-Open up a terminal and go to your crawler's project folder in your local machine::
+It's possible to deploy Docker images with spiders to Scrapy Cloud. To be able to run spiders in custom `Docker`_
+images it's necessary to follow the :ref:`Custom images contract <custom-images-contract>` - a set of requirements
+that image should comply with to be compatible with Scrapy Cloud. The tools for managing and deploying
+Docker images are bundled under ``shub image``.
 
-    $ cd path/to/your/project
+.. _Docker: https://docs.docker.com/
 
-And then run the :ref:`init command <commands-init>`::
-
-    $ shub image init --requirements path/to/requirements.txt
+Deployment
+==========
 
-This will create a Dockerfile for your container including ``requirements.txt`` as a dependency and using `python:2.7 <https://hub.docker.com/r/library/python/>`_ as `the base image <https://docs.docker.com/engine/reference/builder/>`_ for your custom image. If you want to use a different one, pass the ``--base-image`` option, like this::
+This section describes how to build and deploy a custom Docker image to Scrapy Cloud. For all the following steps
+it's assumed that commands are executed at the root directory of your project.
 
-    $ shub image init --base-image scrapinghub/base:12.04
 
-In this case, it will use the image available at https://hub.docker.com/r/scrapinghub/base tagged with ``12.04``.
+1. Add scrapinghub.yml
+----------------------
 
-.. warning:: If you want to use Scrapy in the custom Docker image you may want to include
-    `scrapinghub-entrypoint-scrapy`_ in your requirements.txt file.
-    It is a support layer that passes data from the job to Scrapinghub storage.
-    Otherwise you will need to send data to Scrapinghub storage using `HTTP API`__.
+:ref:`scrapinghub.yml <configuration>` is a Scrapinghub configuration file that defines deploy targets
```

Comment:
It's probably not clear what are the `deploy targets`, maybe just drop this part?
```
is a Scrapinghub configuration file that links local code repository with projects
```